docs/source/cluster_management/kvs-migration.rst

```diff
@@ -0,0 +1,91 @@
+.. _kvs-migration:
+
+=============
+KVS Migration
+=============
+
+.. danger::
+
+    Ensure you have backups before attempting a KVS migration.
+    Double check that `migrateConfig` (the target KVS) is specified correctly, since all tables in that KVS will be dropped.
+
+Overview
+========
+
+Migration of all transactional data from one KVS to another can be performed using the :ref:`AtlasDB CLI<clis>`.
+The source KVS (which contains the data to migrate) should be specified in the ``fromConfig``.
+The target KVS (which we will migrate to) should be specified in the ``migrateConfig``.
+
+There following are AtlasDB tables with special behaviour with respect to KVS migration:
+
+- Atomic tables, as specified in ``AtlasDbConstants.ATOMIC_TABLES``, are internal tables that use check and set and are not transactional.
+
+- Hidden tables, as specified in ``AtlasDbConstants.HIDDEN_TABLES``, are internal tables that are not transactional.
+
+- Cassandra hidden tables, as specified in ``HiddenTables.HIDDEN_TABLES``, are internal tables used by Cassandra KVS and are not transactional.
```

Comment:
Can we rename this one or the one above to not be so ambiguous?

Reply:
This one is actually a private Set, so three times hidden. We can therefore also rename with no fear